src/lib/core/DataModelTypes.h

```diff
@@ -59,6 +63,11 @@ constexpr EndpointId kRootEndpointId    = 0;
 constexpr ListIndex kInvalidListIndex   = 0xFFFF; // List index is a uint16 thus 0xFFFF is a invalid list index.
 constexpr KeysetId kInvalidKeysetId     = 0xFFFF;
 
+constexpr SceneGroupID kGlobalGroupSceneId = CHIP_CONFIG_SCENES_GLOBAL_SCENE_GROUP_ID;
+constexpr SceneIndex kUndefinedSceneIndex  = 0xff;
+constexpr SceneId kUndefinedSceneId        = CHIP_CONFIG_SCENES_TABLE_NULL_INDEX;
+constexpr SceneId kUnusedEndpointId        = CHIP_CONFIG_SCENES_TABLE_UNUSED_ENDPOINT_ID;
```

Comment:
As previously stated, several of these don't need to be configurable, and in fact, may be implementation details for SceneTableImpl.cpp alone.